diff --git a/Makefile b/Makefile
index cb2bd98..bfe9813 100644
--- a/Makefile
+++ b/Makefile
@@ -26,6 +26,6 @@ SYSTEMD_DIR = $(DESTDIR)$(PREFIX)/lib/systemd/system
 .PHONY: install
 
 install:
-	@./find_snapper_config || sed -i 's@^SNAPPER_CONFIG.*@SNAPPER_CONFIG='$(SNAPPER_CONFIG)'@g' bin/$(PKGNAME)
+	@./find_snapper_config || sed -i 's@^SNAPPER_CONFIG=.*@SNAPPER_CONFIG='$(SNAPPER_CONFIG)'@g' bin/$(PKGNAME)
 	@install -Dm755 bin/* -t $(BIN_DIR)/
-	@install -Dm644 ./$(SNAPPER_TEMPLATES)/* -t $(SNAPPER_TEMPLATES)/
+	@install -Dm644 ./$(SNAPPER_TEMPLATES)/* -t $(DESTDIR)/$(SNAPPER_TEMPLATES)/
diff --git a/find_snapper_config b/find_snapper_config
index a04fae3..ee78db7 100755
--- a/find_snapper_config
+++ b/find_snapper_config
@@ -5,7 +5,7 @@ etcdirs="sysconfig default conf.d"
 for x in $etcdirs; do
     d=/etc/$x/snapper
     if [[ -f $d ]]; then
-	    sed -i 's@^SNAPPER_CONFIG.*@SNAPPER_CONFIG='$d'@g' bin/snap-sync
+	    sed -i 's@^SNAPPER_CONFIG=.*@SNAPPER_CONFIG='$d'@g' bin/snap-sync
         exit 0
     fi
 done
